dbms/src/Storages/DeltaMerge/SegmentReadTask.h

```diff
@@ -115,6 +106,8 @@ struct SegmentReadTask
         return input_stream;
     }
 
+    String string() const;
```

Comment:
nit: `toString` maybe a more suitable name
```suggestion
    String toString() const;
```